README.md

```diff
@@ -26,7 +26,8 @@ bx wsk list
 ```
 ## Deploy
 
-Use wskdeploy to deploy using [`manifest.yml`](./manifest.yml)
+Use wskdeploy to deploy using [`manifest.yml`](./manifest.yml).  You can deploy either the
```

Comment:
Use the `wskdeploy` command to deploy using the [`manifest.yml`](./manifest.yml) file.  You can deploy either the